assets/js/modules/tagmanager/components/setup/SetupMain.js

```diff
@@ -50,29 +50,23 @@ export default function SetupMain( { finishSetup } ) {
 	const isDoingGetAccounts = useSelect( ( select ) => select( STORE_NAME ).isDoingGetAccounts() );
 	const isDoingSubmitChanges = useSelect( ( select ) => select( STORE_NAME ).isDoingSubmitChanges() );
 	const hasResolvedAccounts = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getAccounts' ) );
+	const inProgress = useSelect( ( select ) => select( CORE_FORMS ).getValue( FORM_SETUP, 'inProgress' ) );
```

Comment:
How about calling this `submissionInProgress` or something slightly more precise? I think that would clarify the intent of this value better.

Reply:
Sounds good, I'll clarify it.